### ISSUE TITLE
Adding documentation about deployment of frontend.

### ISSUE THREAD

**Issue by Ayush-projects** [Ayush-projects/chords#31]

**Comment by shrishti-c** [comment]
Hey, I'm interested in writing appropriate documentation for deployment of frontend.  Hoping it get assigned.

**Comment by Mayank4git** [comment]
@Ayush-projects I am gssoc'21 participant and looking forward to contribute  to this issue.Please assign this to me

**Comment by Ayush-projects** [comment]
Please add detailed documentation :)

**Comment by Mayank4git** [comment]
@Ayush-projects Is there any deadline for submitting documentation and just help me a bit where to start about project?

**Comment by Ayush-projects** [comment]
Nope, but try to do it fast, so that rest of the participants will get to know info about deployment and it's working. Anyways, doing stuffs fastly can have thier own perks ;)

**Comment by richakumari19** [comment]
Hey , I am participant in GSSOC'21. Can you please assign this to me?

**Comment by Ayush-projects** [comment]
Closing, this issue. Anybody willing to write documentation taking reference of my medium article?

**Comment by Mayank4git** [comment]
I have done half of the documentation using jsdoc tool but you are closing
this

**Comment by Ayush-projects** [comment]
Okay, complete the documentation. Take your time. Adding documentation in level 2 contribution instead of level 1.

**Comment by Mayank4git** [comment]
Sure, will do that.  What about closing it

**Comment by Ayush-projects** [comment]
Will hold it, for 2-3 more days :/
